.travis.yml

```diff
@@ -4,46 +4,94 @@ go_import_path: github.com/skydive-project/skydive
 go:
     - 1.9.1
 
-sudo: required
+sudo: false
+os:
+    - linux-ppc64le
+    - linux
```

Comment:
can you do it only for linux-ppc64le as we already have an job doing for "linux" in our CI. BTW travis is currently disabled, we will re-enable it once your patch merged.

Reply:
here the script used in our CI https://github.com/skydive-project/skydive/blob/master/scripts/ci/create-docker-image.sh